full_snapshot.proto

```diff
@@ -141,6 +143,23 @@ message DatabaseInformation {
   bool collected_local_catalog_data = 11;
 }
 
+message DatabaseStatistic {
+  int32 database_idx = 1;
+
+  uint32 transactions_per_sec = 2; // Transactions per second for this database
+  uint32 min_xid = 3; // Oldest transaction ID for this database
+  uint32 min_mxid = 4; // Oldest multixact ID for this database
```

Comment:
Similar to my note later, I think it would be good to prefix these with `frozen_` and consider making them 64-bit.